packages/SimPKPD/src/pk-pd-tools.ts

```diff
@@ -0,0 +1,165 @@
+/* PK-PD modelling tools.
+ 
+  REFERENCES
+
+    [1] Huixi Zou, et al, Application of Pharmacokinetic-Pharmacodynamic Modeling in Drug Delivery: 
+        Development and Challenges (2020), https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7348046/
+
+    [2] RxODE, https://nlmixrdevelopment.github.io/RxODE/
+
+    [3] W. Wang, et al, A Tutorial on RxODE: Simulating Differential Equation Pharmacometric Models in R (2016),
+        https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4728294/pdf/PSP4-5-03.pdf     
+*/
+
+import * as grok from 'datagrok-api/grok';
+import * as ui from 'datagrok-api/ui';
+import * as DG from 'datagrok-api/dg';
+
+// Wasm-solving tools for PK-PD simulation
+import { simulateOneCompartmentPK, simulateTwoCompartmentPK } from '../wasm/solving-tools';
+
+const DEPOT_T = 'depot(t)';
+const DEPOT = 'Depot';
+const CENTR_T = 'centr(t)';
+const CENTR = 'Central';
+const PERI_T = 'peri(t)';
+const PERI = 'Peripheral';
+const EFF_T = 'eff(t)';
+export const EFFECT = 'Effect';
+const T_COL = 't';
+export const TIME = 'Time [h]';
+export const CENTR_CONC = 'Central concentration';
+const PER_CONC = 'Peripheral concentration';
+const POINTS_COUNTER = 100;
+
+/** Compute concentrations */
+function getConc(centrCol: DG.Column, periCol: DG.Column, V2: number, V3: number): {C2: DG.Column, C3: DG.Column} {
```

Comment:
I'm not sure why are we dong this - is this some sort of a demo dataset? Probably need some comment on top

Reply:
DONE.
Output columns contain data that is evaluated outside wasm-computations, after the main results are obtained.